### DISCUSSION TITLE
Using Themes

### DISCUSSION THREAD

**Discussion by fuzenco** [zed-industries/zed#7178] [Feature Requests]
The Themes doc (https://zed.dev/blog/user-themes-now-in-preview) states there’s a Themes folder in the config folder but I don’t see one. Should we create it manually?

Also, is there any way to access the current theme files in the macOS Finder so we can use a theme as a starting point? I know the doc says we can override/edit the current theme we’re using, but I’d like to learn from a full theme file.

Zed: v0.121.0 (Zed Preview)
OS: macOS 14.3.0
Memory: 16 GiB
Architecture: aarch64

**Comment by krstp** [comment]
Yes, create `theme` folder.

Any theme samples you can find here: https://github.com/zed-industries/zed/tree/main/assets/themes
